src/commands/open-document-settings-panel.ts

```diff
@@ -20,18 +23,24 @@ export const openDocumentSettingsPanel = (name: string, tab = 'Post'): void => {
   // Open Settings tab
   cy.openDocumentSettingsSidebar(tab);
 
-  cy.get('.components-panel__body .components-panel__body-title button')
-    .contains(name)
-    .then($button => {
-      // Find the panel container
-      const $panel = $button.parents('.components-panel__body');
+  // WordPress prior to 5.4 is using upper-case-words for panel names
+  // WordPress 5.3 and below: "Status & Visibility"
+  // WordPress 5.4 and after: "Status & visibility"
+  const ucFirstName = ucFirst(name);
+  const ucWordsName = capitalize(name);
+
+  const panelButtonSelector = `.components-panel__body .components-panel__body-title button:contains("${ucWordsName}"),.components-panel__body .components-panel__body-title button:contains("${ucFirstName}")`;
+
+  cy.get(panelButtonSelector).then($button => {
```

Comment:
Some tabs (Tags, Categories, etc.) does not exist after the DOM is ready. They are added a few moments later after REST calls. In this case, using `cy.get('button').contains(name)` doesn't work: `cy.get()` is called before the tabs appear in DOM, finds other panels, but fails to filter any containing the name.

Using new panelButtonSelector with `button:contains("tab name")` to wait for the elements to appear in DOM.